ci-operator/step-registry/ipi/conf/azure/oidc-creds-deprovision/ipi-conf-azure-oidc-creds-deprovision-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CLUSTER_NAME="${NAMESPACE}-${JOB_NAME_HASH}"
+REGION="${LEASED_RESOURCE}"
+
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+# jq is not available in the ci image...
+#AZURE_AUTH_SUBSCRIPTION_ID="$(jq -r .subscriptionId ${AZURE_AUTH_LOCATION})"
+AZURE_AUTH_SUBSCRIPTION_ID=$(cat ${AZURE_AUTH_LOCATION} | tr -d '{}\"' | tr "," "\n" | grep subscriptionId | cut -d ":" -f2)
```

Comment:
jq is in the upi image if that make things easier

Reply:
I considered using the upi image. However, this step is specifically testing the ccoctl binary from cloud-credential-operator image... which does not have either jq or yq. I considered adding them to the relevant dockerfile, but I don't think we want to add those to the production image. I also considered adding ccoctl to the upi image. Ultimately, I chose to kick the can down the road and use the commands that are available.